Tests/CommandTests/CommandRunnerRaceTests.swift

```diff
@@ -0,0 +1,23 @@
+import Mockable
+import Testing
+@testable import Command
+
+struct CommandRunnerRaceTests {
```

Comment:
@danpalmer I took the opportunity to adjust your test to use Swift Testing instead of XCTest, which we are trying to use less and less.